apps/cookbook/src/app/examples/modal-example/embedded-modal-example/embedded-modal-example.component.ts

```diff
@@ -52,6 +52,8 @@ export class EmbeddedModalExampleComponent implements OnInit {
 
   showModalSizeSelector: boolean;
   selectedModalSize: ModalSize;
+  alertBeforeClose: boolean;
+  disableAlertBeforeClose: boolean = false;
```

Comment:
Consider flipping this so it aligns with the config checkbox (i.e. alert if checked):
```suggestion
```